autogluon/task/tabular_prediction/predictor.py

```diff
@@ -180,7 +180,7 @@ def evaluate_predictions(self, y_true, y_pred, silent=False, auxiliary_metrics=F
     def leaderboard(self, dataset=None, silent=False):
         """
             Output summary of information about models produced during fit() as a pandas DataFrame.
-            Includes information on test and validation scores for all models, model training times and stack levels.
+            Includes information on test and validation scores for all models, model training times, inference times, and stack levels.
```

Comment:
Add description to documentation that describes the columns of the returned dataframe.  In particular, the distinction between the different times.

Reply:
May also want to clarify that these inference times do not account for time needed to load the models, so user may experience additional overhead in practice unless they have already loaded the models.  

More generally, we probably should add another tutorial notebook on reducing inference latency in TabularPrediction.

Reply:
Added description + clarification on model load impacts on value.

Agreed that we should add a tabular notebook on inference latency. Likely after some more PR's that optimize the graph usage further and present the relevant choices to the user in a simple manner (Already have the code, will submit PR soon).